packages/entity-browser/src/modules/detailView/sagas.js

```diff
@@ -21,14 +21,18 @@ export default function* sagas() {
   yield [
     fork(takeLatest, actions.INITIALIZE, initialize),
     fork(takeLatest, actions.LOAD_ENTITY, loadEntity),
-    fork(takeEvery, actions.SUBMIT_FORM, submitForm)
+    fork(takeEvery, actions.SUBMIT_FORM, submitForm),
+    fork(takeEvery, actions.LOAD_RELATION_ENTITIES, loadRelationEntities)
   ]
 }
 
 export function* initialize({payload}) {
   const {entityName, formBase} = payload
 
   yield put(actions.setEntityName(entityName))
+  const entityModel = yield call(fetchModel, entityName)
```

Comment:
Rightnow the model gets loaded in SearchForm and DetailForm, i would refactor that out to root module so its only gets called once.